### PR TITLE
Tweak escape docs

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: doc/filters/escape.rst [hunk 1]
```diff
@@ -8,21 +8,24 @@
...
     The ability to define custom escapers was added in Twig 1.14.0.
 
-The ``escape`` filter escapes a string for safe insertion into the final HTML
-output. It supports different escaping strategies depending on the template
+The ``escape`` filter escapes a string using strategies that depend on the
 context.
 
 By default, it uses the HTML escaping strategy:
 
 .. code-block:: jinja
```
Comment (xabbuh) [inline doc/filters/escape.rst]: `html+jinja` or `html+twig`

File: doc/filters/escape.rst [hunk 2]
```diff
@@ -8,21 +8,24 @@
...
 
-    {{ user.username|escape }}
+    <p>
+        {{ user.username|escape }}
+    </p>
 
 For convenience, the ``e`` filter is defined as an alias:
 
 .. code-block:: jinja
```
Comment (xabbuh) [inline doc/filters/escape.rst]: same here

File: doc/filters/escape.rst (hunk 1 above)
Comment (fabpot) [reply doc/filters/escape.rst]: fixed

File: doc/filters/escape.rst (hunk 2 above)
Comment (fabpot) [reply doc/filters/escape.rst]: fixed

**Review by xabbuh — APPROVED** [review]

**Pull request merged**